lib/puppet/util/log.rb

```diff
@@ -340,10 +340,6 @@ def to_structured_hash
     hash
   end
 
-  def to_pson(*args)
```

Comment:
Are you concerned at all about other places using this? Seems like it might be a stretch but I can possibly imagine someone creating their own log destination that does to_pson. It's unlikely, but this class is also ubiquitous serving as the base for all log events.